mithril-aggregator/src/file_uploaders/dumb_uploader.rs

```diff
@@ -77,4 +85,16 @@ mod tests {
                 .expect("getting dumb uploader last value after a fake download should not fail")
         );
     }
+
+    #[tokio::test]
+    async fn retry_policy_from_file_uploader_trait_should_be_implemented() {
+        let expected_policy = FileUploadRetryPolicy {
+            attempts: 10,
+            delay_between_attempts: Duration::from_millis(123),
+        };
+
+        let uploader: Box<dyn FileUploader> = Box::new(DumbUploader::new(expected_policy.clone()));
```

Comment:
The use of `Box` doesn't seem necessary.
```suggestion
        let uploader = DumbUploader::new(expected_policy.clone());
```

Reply:
Resolved, referred to: https://github.com/input-output-hk/mithril/pull/2244#discussion_r1931943700